### DISCUSSION TITLE
Is it possible to make html text selectable and buttons interactable?

### DISCUSSION THREAD

**Discussion by Tomansion** [cytoscape/cytoscape.js#3302] [Q&A] [answered]
Hello there, 
I've found this example that is almost exactly what I need: https://codesandbox.io/p/sandbox/pedantic-rosalind-7w75t8

![image](https://github.com/user-attachments/assets/0253b493-3ab7-49cf-9644-62797ccb4baa)

This example uses `nodeHtmlLabel` to display some custom HTML and style.

I would like to, from this example, be able to:
- Select text (for copy and paste purposes)
- Click buttons

It seems that Cytoscape mouse drag event takes over the button click event and the text drag. 
I believe that it's very close to working properly because the mouse cursor change when hovering the HTML text, and the button style changes on click.

Has someone figured out how to make is work?

I've tried to play with z-index without results.
Thanks.

**Comment by TonyIvanova** [comment, marked as answer]
@Tomansion You have a syntax error :-) 
It should be onclick instead of @click
```js
  <button onclick="alert('hello')">This is a button</button>
```

**Comment by Tomansion** [comment]
It's working indeed, thanks :man_facepalming: .
This open's up some new fun interactions.
